src/student/student.module.ts

```diff
@@ -11,7 +11,9 @@ import { UserStudentSchema } from './user-student.schema';
         {
           name: 'users',
           schema: UserSchema,
-          discriminators: [{ name: 'userStudent', schema: UserStudentSchema }],
+          discriminators: [
+            { name: UserStudent.name, schema: UserStudentSchema },
```

Comment:
`UserStudent.name` what is it? it's suppose to be a string. is it derive from ClassName + s or es.